app/src/lib/stores/app-store.ts

```diff
@@ -3279,7 +3279,13 @@ export class AppStore extends TypedBaseStore<IAppState> {
       const defaultRemote = await getDefaultRemote(repository)
       // TODO: I think we could skip this fetch if we know that we have the branch locally
```

Comment:
I think this comment can be rewritten to drop the `TODO` and describe the intent here